apps/src/studio/studio.js

```diff
@@ -153,6 +154,16 @@ function loadLevel() {
   // protagonistSpriteIndex was originally mispelled. accept either spelling.
   Studio.protagonistSpriteIndex = level.protagonistSpriteIndex || level.protaganistSpriteIndex;
 
+  switch (level.customGameType) {
+    case 'Big Game':
+      Studio.customGame = new BigGameInfo();
```

Comment:
We could eventually add a jsdoc `@interface` type similar to the `AudioPlayer` dealie here: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/assets/javascripts/soundEffects.js#L1